Clean logger. Add empty logger. require proper setup (#51)
uniflow-kt/uniflow-kt: Changelog.md

## 0.11.x

### 0.11.7

> - `added` Logger clean & requires setup, else would throw IllegalStateException - use `UniFlowLogger.init(...)`,  `UniFlowLogger.default()` or `UniFlowLogger.empty()`

Reviewer: ```suggestion
- `added` an empty logger (`EmptyLogger`, logs nothing): use `UniFlowLogger.init(EmptyLogger())` or `UniFlowLogger.empty()`
```
